configs/polad.yaml.sample

```diff
@@ -2,12 +2,16 @@ global:
   pcep:
     address: "127.0.0.1"
     port: 4189
-  grpc:
+  grpc-server:
     address: "127.0.0.1"
     port: 50052
   log:
     path: "/var/log/pola/"
     name: "polad.log"
+  ted:
+    enable: true
+    update-tool: "gobgp"
```

Comment:
The name "update-tool" feels like you need to use a tool to get the TED, so names like "input", "source", or "bgp-daemon" might be better.